### PR TITLE
Update Remnant Cognizance 28

### PR DESCRIPTION
**Pull request by Aneutronic**
## Summary
In "Remnant: Cognizance 28," the standard cargo units are referred to as "cubic meters." As I believe "tons" is the standard, I changed it to match. I also added detail to the short mission prompt displayed on the mission screen to include how much cargo and how many bunks are needed on the active ship.

### CODE REVIEW

**Comment by Zitchas** [comment]
A note, that while tons may be standard elsewhere; the Remnant are metric, and I would this prefer to not replace references to cubic metres as a measure of volume.

**Comment by Aneutronic** [comment]
Okay. I just assumed that the same units would be used everywhere to avoid confusion. It is not intuitive to players that 1 unit of cargo can be both a "ton" and a "cubic meter." Besides, I just assumed they were using metric tons.

**Comment by warp-core** [comment]
I agree that `cubic metres` is not necessarily intuitively equivalent to `tons`, especially when one is a unit of volume and the other is a unit of weight/mass.
While we're on the topic, if the Remnant are so metric, why do they mix base 16 values with the base 10 units of metric? That's not done here, but it is elsewhere.

**Comment by Aneutronic** [comment]
@Hecter94 Why did you remove the Ember Waste label? This discussion is entirely centered around the units used in the dialog inside Ember Waste campaign missions.

**Comment by warp-core** [comment]
> @Hecter94 Why did you remove the Ember Waste label? This discussion is entirely centered around the units used in the dialog inside Ember Waste campaign missions.

There has been some contention around the use of that label.

**Comment by Hecter94** [comment]
> @Hecter94 Why did you remove the Ember Waste label? This discussion is entirely centered around the units used in the dialog inside Ember Waste campaign missions.

As Warp said. That label is currently not used on this repo. As such, I removed it to avoid confusion.

File: data/remnant/remnant 2 cognizance.txt [hunk 1]
```diff
@@ -1245,7 +1245,7 @@ mission "Remnant: Cognizance 28"
...
+			`Your commlink springs to life as you enter final approach to Caelian, revealing Torza in a flurry of motion. He focuses momentarily on the camera. "Captain <first>, and flying a ship with a substantial lifting capacity, excellent! The Korath have discovered one of our forward operating bases, and we need help evacuating it now. It is located on Baianus in the Parca system. There are 20 people and 280 tons of equipment to retrieve. The Korath have set up a blockade, so you will need something reasonably fast to get through it. You can fight them or not, as you wish, but the primary task is evacuating the outpost. Embers speed your flight, Captain."`
```
Comment (Zitchas) [inline data/remnant/remnant 2 cognizance.txt:1248]:
```suggestion
			`Your commlink springs to life as you enter final approach to Caelian, revealing Torza in a flurry of motion. He focuses momentarily on the camera. "Captain <first>, and flying a ship with a substantial lifting capacity, excellent! The Korath have discovered one of our forward operating bases, and we need help evacuating it now. It is located on Baianus in the Parca system. There are 20 people and 280 cubic meters of equipment to retrieve. The Korath have set up a blockade, so you will need something reasonably fast to get through it. You can fight them or not, as you wish, but the primary task is evacuating the outpost. Embers speed your flight, Captain."`
```

As previously mentioned, I would prefer to leave the dialog mention of cubic meters in place.

**Review by Zitchas — CHANGES_REQUESTED** [review]
Thanks for suggestion the change to the mission description. That's a useful change that will help people keep track of what the goal is.

I would prefer that the reference to cubic meters remain, however.

Thanks!

**Review by Zitchas — APPROVED** [review]
Looks good. Thanks!

**Pull request merged**